Agreements/External/YY-D_11.txt

```diff
@@ -0,0 +1,2 @@
+FIR_COPX:*:*:RAVLO:EGSH:*:EDYY Delta:London S11:*:30000:5RAVLO
+FIR_COPX:*:*:RAVLO:EGNJ:*:EDYY Delta:London S11:*:32000:5RAVLO
```

Comment:
Looking at our other COPX setups, it looks like we have the number after the waypoint name in the last bit. So in this case it would be RAVLO5 instead of 5RAVLO

Reply:
Eeeek! Yeah. Swapping now :D